### ISSUE TITLE
top(1) compatibility: capital-V hotkey for tree view

### ISSUE THREAD

**Issue by Explorer09** [hishamhm/htop#378]
The top program from procps-ng uses 'V' (capital-V) for toggling what is called "tree view" in htop.
However htop uses F5 and lowercase-t for this.
Is there a need to follow them and change the hotkey?

(Feel free to tag 'wishlist' for this issue report.)
Reference: http://man7.org/linux/man-pages/man1/top.1.html (Search for keyword "Forest view")

**Comment by hishamhm** [comment]
Why would we follow them, if htop has had tree view for years before they copied the feature from us? :)

When I first released htop, I made sure to make htop feel familiar to top users following the conventions of the day such as the column titles and many shortcuts (M, P, etc.), but nowadays I don't feel the need to keep trying to match top (also because there are several tops out there). I'm happy to say that, 12 years on, we already have enough tradition of our own to have our own shortcuts. :)
